### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by NateyLB**
### Description
Successfully fetched dummy data from our API
Integration of Google Maps API
Creation of Map Markers from dummy data
Map is centered on the US and is responsive
Removed console logs

<p> https://trello.com/c/Nceq4oYJ </p>
<p> [x] Ran locally and passed all tests </p>

### CODE REVIEW

File: src/components/pages/Incidents/Map/Marker.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Marker = props => {
+  return (
+    <div
+      className="marker-container"
+      style={{
+        height: '5px',
+        width: '5px',
+        backgroundColor: 'red',
```
Comment (BrannanC) [inline src/components/pages/Incidents/Map/Marker.js]: Is there a reason for having these styles inline?

Comment (NateyLB) [reply src/components/pages/Incidents/Map/Marker.js]: Just leaving them online for now until the organization for styling is more clear. Will move into a less style sheet soon.

Comment (BrannanC) [reply src/components/pages/Incidents/Map/Marker.js]: Let's get that organization figured out and apply more styling before we ship this feature.

File: src/styles/css/index.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  align-items: center;
+  width: 100%;
+}
+.consent-container .consent-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  width: 90%;
+}
+@media (min-width: 768px) {
```
Comment (BrannanC) [inline src/styles/css/index.css:143]: Let's make variables for our breakpoints

File: src/styles/css/index.css [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+}
+.marker {
+  height: 5px;
+  width: 5px;
+  background-color: red;
+  border-radius: 20px;
```
Comment (BrannanC) [inline src/styles/css/index.css]: Border radius should be a variable as well

File: src/styles/less/landing.less [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    height: 100vh;
+    .flexDisplay(row, center, center);
+    width: 100%;
+
+    .consent-card{
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        width: 90%;
+        @media @medium{
```
Comment (BrannanC) [inline src/styles/less/landing.less:11]: Are these the same as the hard coded media queries?

File: src/styles/css/index.css (hunk 3 above)
Comment (NateyLB) [reply src/styles/css/index.css]: border-radius here is used to make the marker a circle, one-time use.

**Review by ph5500 — APPROVED** [review]
Changes are working as expected. We will need to make changes if/when we decide to add clustering.

**Pull request merged**